fog-digitalocean.gemspec

```diff
@@ -7,6 +7,8 @@ Gem::Specification.new do |s|
   s.specification_version = 2 if s.respond_to? :specification_version=
   s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
 
+  s.required_ruby_version = '>= 2.0.0'
```

Comment:
I think we could leave this for another version bump and make this change go smoothly. WDYT?

Reply:
I guess we can leave this out since someone could in theory pin to an old version of the deps that would still work on Ruby < 2